### DISCUSSION TITLE
aarch64 Ubuntu?

### DISCUSSION THREAD

**Discussion by da0ist** [rustdesk/rustdesk#1045] [Q&A]
I'm currently running NoMachine for Pi on my aarch64 Ubuntu machine. I'd like to try rustdesk. The pi version is packaged for armhf?

**Comment by WangFengtu1996** [comment]
I also need a aarch64 version.

**Reply by goodluck1982** [reply]
me too

**Reply by sdjeny** [reply]
+1

**Reply by rustdesk** [reply]
PR welcome, please add aarch64 Ubuntu 18 to nightly build CI and test , https://github.com/rustdesk/rustdesk/blob/master/.github/workflows/flutter-nightly.yml
